CHANGELOG.next.toml

```
@@ -192,3 +192,9 @@ message = "Fix aws-sigv4 canonical request formatting fallibility."
 references = ["smithy-rs#1656"]
 meta = { "breaking" = false, "tada" = false, "bug" = true }
 author = "ysaito1001"
+
+[[smithy-rs]]
```

The `aws-sigv4` package is part of the SDK rather than part of smithy-rs (at least as far as the changelog goes).
```suggestion
[[aws-sdk-rust]]
```